test/WorkOSTests/Services/DirectorySync/DirectorySyncServiceTest.cs

```diff
@@ -61,6 +61,8 @@ public DirectorySyncServiceTest()
                 ExternalKey = "external-key",
                 EnvironmentId = "environment_123",
                 OrganizationId = "organization_123",
+                CreatedAt = "10-04-19 12:00:17",
```

Comment:
Hmm, these still don't look totally correct.

The API returns dates in ISO 8601 format, so they should look like this: `2021-07-26T18:55:16.072Z`

Reply:
Got it, wasn't totally sure of the format. I'll get it changed.